priam/src/main/java/com/netflix/priam/PriamServer.java

```diff
@@ -151,8 +152,8 @@ else if (UpdateSecuritySettings.firstTimeUpdated)
         scheduler.addTaskWithDelay(CassandraMonitor.JOBNAME,CassandraMonitor.class, CassandraMonitor.getTimer(), CASSANDRA_MONITORING_INITIAL_DELAY);
         
         // Start the snapshot backup schedule - Always run this. (If you want to
-        // set it off, set backup hour to -1)
-        if (config.getBackupHour() >= 0 && (CollectionUtils.isEmpty(config.getBackupRacs()) || config.getBackupRacs().contains(config.getRac())))
+        // set it off, set backup hour to -1) or set backup cron to "0 0 5 31 2 ?" (Feb 31)
```

Comment:
Instead of having it scheduled for Feb 31, not scheduling with -1 is a good idea